adhocracy4/comments/models.py

```diff
@@ -39,6 +39,9 @@ class Comment(base.UserGeneratedContentModel):
     is_moderator_marked = models.BooleanField(default=False)
     # used in moderation dashboard, indicates if comment is marked as read by moderator
     is_reviewed = models.BooleanField(default=False)
+    project = models.ForeignKey(
+        Project, on_delete=models.CASCADE, blank=True, null=True
```

Comment:
Do we really want blank here? Probably every comment should have the project set, right?

Reply:
yes and no, some comments don't have the module set, so getting the project in the migration fails. I'm not sure why but maybe if a module or project is deleted? Do we keep the comments in that case? At least we probably don't want the cascade here I think?

Reply:
@Rineee @fuzzylogic2000 last thing we should check: do we want the cascade there? I guess we don't (if it should be deleted it will happen it will come from somewhere else) ?

Reply:
Hm, yeah, probably it will be deleted anyway then.. but still we could keep it here, I would say? What do you think @fuzzylogic2000 ?

Reply:
Sure, I am also fine with `SET_NULL` :)